packages/itwinui-css/src/searchbox/searchbox.scss

```diff
@@ -1,32 +1,17 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
 
-.iui-expandable-searchbox {
-  inline-size: var(--_iui-input-min-height);
-  transition: width var(--iui-duration-1) ease-in-out;
-
-  &:has(.iui-searchbox-open-button:focus-visible) {
-    outline: 2px solid var(--_iui-input-accent-border-color);
-    outline-offset: -2px;
-  }
+@use '../utils/icon' as *;
+@use 'sass:map';
```

Comment:
These look accidentally added/leftover?

Reply:
Yes, removed in [`d853b40` (#1888)](https://github.com/iTwin/iTwinUI/pull/1888/commits/d853b404717fd6ed0e2c8f2f281f704342b6c7a8).